langchain4j-core/src/main/java/dev/langchain4j/store/embedding/EmbeddingStore.java

```diff
@@ -79,4 +79,31 @@ default List<EmbeddingMatch<Embedded>> findRelevant(Embedding referenceEmbedding
      * ranging from 0 (not relevant) to 1 (highly relevant).
      */
     List<EmbeddingMatch<Embedded>> findRelevant(Embedding referenceEmbedding, int maxResults, double minScore);
+
+    /**
+     * Finds the most relevant (closest in space) embeddings to the provided reference embedding.
+     * By default, minScore is set to 0, which means that the results may include embeddings with low relevance.
+     * @param originText the origin text of user input
+     * @param referenceEmbedding The embedding used as a reference. Returned embeddings should be relevant (closest) to this one.
+     * @param maxResults         The maximum number of embeddings to be returned.
+     * @return A list of embedding matches.
+     * Each embedding match includes a relevance score (derivative of cosine distance),
+     * ranging from 0 (not relevant) to 1 (highly relevant).
+     */
+    default List<EmbeddingMatch<Embedded>> findRelevant(Object memoryId,String originText,Embedding referenceEmbedding, int maxResults) {
```

Comment:
why do you need to pass `originText` into `EmbeddingStore`?

Reply:
it is useless ,if i have memoryId,i will remove